tools/generate-doc-preview-index.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+#
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0.
+#
+
+HEAD_COMMIT_SHA=$(git rev-parse HEAD)
+cd $(git rev-parse --show-toplevel)/target/doc
+
+{
+    echo '<!doctype html>'
+    echo '<html>'
+    echo '<head>'
+    echo '  <metadata charset="utf-8">'
+    echo "  <title>Doc preview for ${HEAD_COMMIT_SHA}</title>"
+    echo '</head>'
+    echo '<body>'
+    echo "  <h1>Doc preview for ${HEAD_COMMIT_SHA}</h1>"
+} > index.html
+
+doc_link () {
+    local package=$1
+    if [[ -d ${package} ]]; then
+        echo "    <li><a href='${package}/index.html'>${package}</a></li>" >> index.html
```

Comment:
```suggestion
        echo "  <li><a href='${package}/index.html'>${package}</a></li>" >> index.html
```

The rest of the indents are two spaces.

Reply:
Actually, this one was intentionally at four spaces since it's nested inside a `<ul>`.